### DISCUSSION TITLE
VIM activated but not working

### DISCUSSION THREAD

**Discussion by Bu1aKa** [Mr-MIBonk/M.I.B._More-Incredible-Bash#56] [Q&A]
Hello together,

i installed the MIB successfull
1st. Step; Backup
2st. AIO Patch 
3rd. VIM set to 199 -> but its not working. If I start driving the secure information of the MMI appears on the Screen.

to be honest:
ist tryd it already with VCDS online and with odbeleven to unlock VIM without success

i can send the logs, if needed

kind regards
Bu1

**Comment by Bu1aKa** [comment]
[MU1177-YJL-26010.02.186969018A.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/9953905/MU1177-YJL-26010.02.186969018A.log)

RCC File is to big for upload

**Comment by harman-f** [comment]
```
MMX-vim---->
VIM on Unit: C7010F0AFF00000000000301FF00000003010F050F05FF0000053135AF40
Position->0 - HEX-Value->06

Generate crc16/CCITT-False checksum............................... and the Winner is: 478E

VIM is now patched on Byte 0 with 6 km/h (0x06)
Reboot Unit...
```

Looking at your 1st LOG entry, your unit already came "patched"
hex C7 is already 199km/h

Well, anyway M.I.B will do it right, change the 1st two bytes and calculate new CRC.
It can not check if the dataset is somehow broken...

There are also 1-2 FW, which have broken VIM from factory.
Not 100% sure, but I think MU1177-MHI2Q_ER_AUG22_P4212 could be one of them.
